src/actions/receive-profile.js

```diff
@@ -1014,7 +1014,9 @@ export function getProfilesFromRawUrl(
     switch (dataSource) {
       case 'from-addon':
         shouldSetupInitialUrlState = false;
-        await dispatch(retrieveProfileFromAddon());
+        // It doesn't need to be synchronous because we don't need the profile
+        // data to setup the url for addon.
```

Comment:
nipicking :) I find that the word "synchronous" is not very precise here.
What about:
```
We don't need to await the result because there's no url upgrading
when retrieving the profile from the addon and so we don't need to
wait for the process. Moreover we don't want to wait for the end of
symbolication and rather want to show the UI as soon as we get
the profile data.
```

Reply:
Works for me, changed :)